packages/access-api/test/provider-add.test.js

```diff
@@ -1,282 +1,282 @@
 import {
   assertNotError,
-  createTesterFromContext,
-  createTesterFromHandler,
+  registerSpaces,
   warnOnErrorResult,
 } from './helpers/ucanto-test-utils.js'
 import * as principal from '@ucanto/principal'
-import * as provider from '@web3-storage/capabilities/provider'
 import * as assert from 'assert'
-import { createProviderAddHandler } from '../src/service/provider-add.js'
-import { context } from './helpers/context.js'
 import * as ucanto from '@ucanto/core'
 import * as Ucanto from '@ucanto/interface'
 import { Access, Provider } from '@web3-storage/capabilities'
 import * as delegationsResponse from '../src/utils/delegations-response.js'
-import { createProvisions } from '../src/models/provisions.js'
 import { NON_STANDARD } from '@ipld/dag-ucan/signature'
-import { createEmail } from './helpers/utils.js'
-
-for (const providerAddHandlerVariant of /** @type {const} */ ([
-  {
-    name: 'handled by createProviderAddHandler',
-    ...(() => {
-      const spaceWithStorageProvider = principal.ed25519.generate()
-      const service = {
-        did: () => /** @type {const} */ ('did:web:web3.storage'),
-      }
-      const provisions = createProvisions(service.did())
-      return {
-        spaceWithStorageProvider,
-        provisions,
-        ...createTesterFromHandler(() =>
-          createProviderAddHandler({
-            provisions,
-          })
-        ),
-      }
-    })(),
-  },
-])) {
-  describe(`provider/add ${providerAddHandlerVariant.name}`, () => {
-    it(`can be invoked by did:key`, async () => {
```

Comment:
Note: This test was removed because it was not testing a right thing and was only passing because it did not used validator.